app/src/main/res/menu/download_log.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:custom="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/refresh_item"
+        android:title="@string/refresh_label"
+        android:menuCategory="container"
+        custom:showAsAction="always"
+        android:icon="?attr/navigation_refresh"/>
```

Comment:
Hmm. I think users could confuse this with the normal refresh button. Do you have another idea for an icon? If not, I would vote to move this to the overflow menu (by setting `showAsAction="never"`), where it is shown as text.